avro/src/main/java/com/fasterxml/jackson/dataformat/avro/ser/AvroDateDateSerializer.java

```diff
@@ -0,0 +1,29 @@
+package com.fasterxml.jackson.dataformat.avro.ser;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class AvroDateDateSerializer extends JsonSerializer<Date> {
+  public static final JsonSerializer<Date> INSTANCE = new AvroDateDateSerializer();
+  private static final TimeZone UTC = TimeZone.getTimeZone("UTC");
+  private static final long MILLIS_PER_DAY = 24 * 60 * 60 * 1000;
+
+  @Override
+  public void serialize(Date value, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+    Calendar calendar = Calendar.getInstance(UTC);
```

Comment:
to avoid Thai Buddhist or Japanese Imperial calendars, it may be safer to explicitly instantiate a GregorianCalendar here

Reply:
alternatively you can convert via Instant and ZonedDateTime, avoiding Calendar altogether